src/nvim/api/autocmd.c

```diff
@@ -503,28 +561,33 @@ Integer nvim_create_autocmd(uint64_t channel_id, Dict(create_autocmd) *opts, Err
   return autocmd_id;
 }
 
-/// Delete an autocmd by ID. Autocmds only return IDs when created
-/// via the API.
+/// Delete an autocommand by id.
 ///
-/// @param id Integer The ID returned by nvim_create_autocmd
+/// @param id Integer The id returned by nvim_create_autocmd
+/// @see nvim_create_augroup
+/// @see nvim_del_augroup_by_name
 void nvim_del_autocmd(Integer id)
   FUNC_API_SINCE(9)
 {
   autocmd_delete_id(id);
 }
 
-/// Create or get an augroup.
+/// Create or get an autocommand group. See |autocmd-groups|.
 ///
-/// To get an existing augroup ID, do:
+/// To get an existing augroup id, do:
 /// <pre>
-///     local id = vim.api.nvim_create_augroup({ name = name, clear = false });
+///     local id = vim.api.nvim_create_augroup({
+///       name = "MyGroup",
+///       clear = false
+///     });
 /// </pre>
 ///
-/// @param opts Parameters
-///                 - name (string): The name of the augroup
-///                 - clear (bool): Whether to clear existing commands or not.
-//                                  Defaults to true.
-///                     See |autocmd-groups|
+/// @param opts Dictionary Parameters
+///                 - name: (string) The name of the augroup
+///                 - clear: (bool), defaults to true. Clear existing commands if the group already
+///                 exists. See |autocmd-groups|.
+/// @return Integer The id of the created augroup.
```

Comment:
I think usually we don't include the return type.

```suggestion
/// @return The id of the created augroup.
```